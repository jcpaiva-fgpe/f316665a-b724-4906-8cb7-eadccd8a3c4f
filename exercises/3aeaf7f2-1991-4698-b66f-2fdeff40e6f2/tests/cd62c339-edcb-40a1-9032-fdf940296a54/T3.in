10
A2
B13
C10
D1
E1
F3
G12
H1
K6
W49
