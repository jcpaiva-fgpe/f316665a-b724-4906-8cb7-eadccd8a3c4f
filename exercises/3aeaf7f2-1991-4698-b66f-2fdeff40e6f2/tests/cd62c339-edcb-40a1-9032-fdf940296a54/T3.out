AA
BBBBBBBBBBBBB
CCCCCCCCCC
D
E
FFF
GGGGGGGGGGGG
H
KKKKKK
WWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWW
